src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -79,4 +95,485 @@ protected JacobianMatrix createJacobianMatrix(EquationSystem equationSystem, Vol
         equationSystem.updateEquations(x);
         return new JacobianMatrix(equationSystem, matrixFactory);
     }
+
+    protected void fillRhsSensitivityVariable(LfNetwork lfNetwork, EquationSystem equationSystem, List<SensitivityFactorGroup> factorGroups, Matrix rhs) {
```

Comment:
These three functions should be moved in order to have a more understandable ranking in the file. The ranking could be simply the calling order.